README.md

```diff
@@ -92,83 +99,106 @@ The script `hack/test/local_test.sh` can be used to automate local testing by cr
 
-```
+```shell
 oc create configmap certman-operator \
     --from-literal=default_notification_email_address=foo@bar.com
 ```
 
 ### Certman Operator Secrets
 
-A [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) is used to store the Let's Encrypt account url and keys.
+There are two [secrets](https://kubernetes.io/docs/concepts/configuration/secret/) required for certman-operator to function.
 
+1. `lets-encrypt-account` - This secret is used to store the Let's Encrypt account url and keys.
+
+```bash
+# To fetch the "lets-encrypt-account" secret for a cluster on the Hive shard.
+oc -n certman-operator get secret lets-encrypt-account -oyaml
 ```
- oc create secret generic lets-encrypt-account-staging \
-    --from-file=private-key=production-private-key.pem \
-    --from-file=account-url=production-account.txt
- oc create secret generic lets-encrypt-account-production \
-    --from-file=private-key=staging-private-key.pem \
-    --from-file=account-url=staging-account.txt
+
+For testing purposes:
+
+```bash
+# On the staging cluster:
+oc -n certman-operator create secret generic lets-encrypt-account \
+    --from-file=private-key=private-key.pem \
+    --from-file=account-url=account.txt
 ```
 
+2. `aws` or `gcp` - Based on which platform is being used (AWS or GCP), this is the secret which contains the cloud platform credentials of the account of the target cluster.
+
+```bash
+# To fetch the "aws" secret for a cluster on the Hive shard.
+NAMESPACE=$(oc get cd -A | grep -i $CLUSTERNAME | awk '{ print $1 }')
+oc -n $NAMESPACE get secret aws -oyaml
+```
+
+For testing purpose:
+
+```bash
+# To create the "aws" secret on staging cluster for testing.
+oc -n certman-operator create secret generic aws --from-literal=aws_access_key_id=XXX 
+--from-literal=aws_secret_access_key=YYYY 
+```
+
+*NOTE:*
+
+- *The 'aws' secret for AWS platform will be required for only non-STS clusters. The STS clusters won't have this secret.*
+
+- *For testing purposes, both the secrets (i.e lets-encrypt-account secret and aws/gcp platform credential secret) can be found on the Hive shard of the staging cluster.*
+
 ### Custom Resource Definitions (CRDs)
 
 #### Create Hive CRDs
 
-```
+```shell
 git clone git@github.com:openshift/hive.git
 oc create -f hive/config/crds
 ```
 
 #### Create Certman Operator CRDs
 
-```
-oc create -f https://raw.githubusercontent.com/openshift/certman-operator/master/deploy/crds/certman.managed.openshift.io_certificaterequests_crd.yaml
+```shell
+oc create -f https://raw.githubusercontent.com/openshift/certman-operator/master/deploy/crds/certman.managed.openshift.io_certificaterequests.yaml
 ```
 
 ### Run Operator From Source
 
-```
-operator-sdk run --local
```

Comment:
Why are you moving away from operator-sdk?

Reply:
The new version of `operator-sdk` doesn't support this flag so an alternative `make run` is added instead. For more context, could refer to [issue](https://github.com/operator-framework/operator-sdk/issues/4158#issuecomment-720139791) and [documentation](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.0.0/#cli-changes).

Reply:
Thanks Dustin. Have added the command to run locally instead for now.

Reply:
In near future, will add it to boilerplate instead as per our discussion.